bin/run.sh

```diff
@@ -34,8 +34,13 @@ elif [ ! -e Cargo.lock ]; then
     echo "WARNING: student did not upload Cargo.lock. This may cause build errors." | tee -a "$output_path/results.out"
 fi
 
+release=""
+if jq --exit-status '.custom?."test-in-release-mode"?' "$solution_path"/.meta/config.json; then
+    release="--release"
+fi
+
 cargo +nightly test \
-    --offline \
+    --offline $release \
```

Comment:
```suggestion
    --offline \
    $release \
```

All the other flags get their own line.